fvm/src/call_manager/default.rs

```diff
@@ -440,6 +468,14 @@ where
     where
         K: Kernel<CallManager = Self>,
     {
+        let from_actor = self.state_tree().get_actor(from)?.ok_or_else(Err(
+            syscall_error!(NotFound; "sender does not exist: {}", to).into(),
+        ))?;
+
+        if self.builtin_actors().is_embryo_actor(from_actor.code) {
+            self.transform_into_eoa(from, from_actor)?;
+        }
```

Comment:
We shouldn't do this inside the call manager as it only applies to external sends. Can we put this in the executor before we call into the callmanager?